### PR TITLE
Fixed typo in documentation

### PR DESCRIPTION
**Pull request by CaselIT**

### CODE REVIEW

**Comment by CaselIT** [comment]
Had to sign the License Agrreement. I signed it now.

File: modules/angular2/src/core/di/opaque_token.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {CONST} from 'angular2/src/facade/lang';
  * var t = new OpaqueToken("value");
  *
  * var injector = Injector.resolveAndCreate([
- *   provide(t, {useValue: "providedValue"})
+ *   provide(t, {useValue: "bindingValue"})
  * ]);
  *
  * expect(injector.get(t)).toEqual("bindingValue");
```
Comment (todoubaba) [inline modules/angular2/src/core/di/opaque_token.ts]: toEqual("bindingValue") => toEqual("providedValue") ?

Comment (CaselIT) [inline modules/angular2/src/core/di/opaque_token.ts]: I've used the same code as in the provided example 
http://plnkr.co/edit/Ys9ezXpj2Mnoy3Uc8KBp?p=preview

**Comment by IgorMinar** [comment]
closed by 9cb6dbbbab4f0ce1ea48f9c983c1f3673f0def63

**Pull request closed without merging**